src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -63,16 +63,28 @@ class ObjectsAPIRegistration(BasePlugin):
     def register_submission(
         self, submission: Submission, options: dict
     ) -> Dict[str, Any]:
+        """Register a submission using the ObjectsAPI backend
+
+        The creation of submission documents (report, attachment, csv) makes use of ZGW
+        service functions (e.g. :func:`create_report_document`) and involves a mapping
+        (and in some cases renaming) of variables which would otherwise not be
+        accessible from here. For example, 'vertrouwelijkheidaanduiding' must be named
+        'docVertrouwelijkheidaanduiding' because this is what the ZGW service functions
+        use."""
+
         config = ObjectsAPIConfig.get_solo()
         config.apply_defaults_to(options)
 
+        options["auteur"] = options.get("auteur", "Aanvrager")
```

Comment:
```suggestion
        options["auteur"] = options.get("auteur") or "Aanvrager"
```
please add tests where `auteur` is an empty string